src/main/java/io/github/thebusybiscuit/hotbarpets/HeadTextures.java

```diff
@@ -0,0 +1,68 @@
+package io.github.thebusybiscuit.hotbarpets;
+
+/**
+ * This class holds the head hashes for all head textures.
+ *
+ * @author WalshyDev
+ */
+public enum HeadTextures {
```

Comment:
Shouldn't enums be singular and not plural?
Also I think `PetTexture` may be a better name here since we already got HeadTexture in Slimefun.

```suggestion
public enum PetTexture {
```